modules/examiner/php/NDB_Menu_Filter_Form_examiner.class.inc

```diff
@@ -60,23 +60,25 @@ class NDB_Menu_Filter_Form_Examiner extends NDB_Menu_Filter_Form
 
         if ($user->hasPermission('examiner_multisite')) {
                 $query = " 
-                        FROM 
-                         examiners e LEFT JOIN psc ON 
-                        (e.centerID=psc.CenterID) 
-                        LEFT JOIN certification c ON 
-                        (c.examinerID=e.examinerID and c.pass = 'certified') 
-                        LEFT JOIN test_names tn ON (tn.ID = c.testID) WHERE 1=1 ";
+                        FROM examiners e 
+                            LEFT JOIN examiners_psc_rel epr ON 
+                                (epr.examinerID=e.examinerID)
+                            LEFT JOIN psc ON (epr.centerID=psc.CenterID) 
+                            LEFT JOIN certification c ON 
+                                (c.examinerID=e.examinerID and c.pass = 'certified') 
+                            LEFT JOIN test_names tn ON (tn.ID = c.testID) 
+                        WHERE 1=1 ";
         } else {
 
               $query = "
-                        FROM 
-                         examiners e LEFT JOIN psc ON 
-                        (e.centerID=psc.CenterID) 
-                        LEFT JOIN certification c ON 
-                        (c.examinerID=e.examinerID and c.pass = 'certified') 
-                        LEFT JOIN test_names tn ON (tn.ID = c.testID)
-                        WHERE e.centerID =".$centerID." ";
-
+                        FROM examiners e 
```

Comment:
given the new SQL architecture full name should be a unique key and thus obey the functional dependency restrictions of MySQL 5.7... I would need someone to test this however since I'm on 5.7